### DISCUSSION TITLE
Want to use fabric to analyze huge code database

### DISCUSSION THREAD

**Discussion by colekimballl** [danielmiessler/Fabric#665] [General]
Any ideas on how to do this efficiently?

**Comment by xeniode** [comment]
Might start with ctags

**Comment by patonw** [comment]
What specific goals do you have in mind? "Analyze" is a pretty broad term. My guess is that you'll get more useful information operating on the documentation/comments than code itself.
Also, for working with huge _, a tool with strong support for splitting/reducing data like langchain might be a better starting point to do pre-processing/indexing. You might then use fabric to interact with the intermediate results, depending on how they're represented.
